composer.json

```diff
@@ -7,6 +7,7 @@
     "require": {
         "php": ">=5.6.4",
         "intervention/image": "^2.4",
+        "laracasts/flash": "^3.0",
```

Comment:
this is list and delete module, the pull  https://github.com/AT-PHPIntership/food-market-order/pull/72/files
is create module, it different!